lib/db.js

```diff
@@ -52,7 +52,12 @@ module.exports = (
   }
 
   function DB(options) {
-    this.pool = new Pool(options.url)
+    let pooleeOptions = {}
+    if (config && config.db && config.db.poolee) {
+      pooleeOptions = config.db.poolee
+    }
+
+    this.pool = new Pool(options.url, pooleeOptions)
```

Comment:
This is fugly, but preserves the way `DB.connect` has been used in multiple tests and other places.